ui/components/app/confirm/info/row/section.tsx

```diff
@@ -8,14 +8,17 @@ import {
 export type ConfirmInfoSectionProps = {
   children: React.ReactNode | string;
   noPadding?: boolean;
+  dataTestId?: string;
```

Comment:
For consistency, replace by `'data-testid`.

Reply:
in the component itself it is 'data-testid', this is just the props, do you mean it should be 'data-testid' in the prop as well?

Reply:
yes.. it should be `data-testid` in the props as well. This ensures consistency, allowing us 
to render the `ConfirmInfoSection` component and set the `data-testid` like this:

```js
<ConfirmInfoSection data-testid="VALUE" />
```

vs

```js
<ConfirmInfoSection dataTestId="VALUE" />
```

Example of a similar implementation: https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/confirm/info/row/currency.tsx#L16

Reply:
done

Reply:
great catch! It seems we pass the value as `dataTestId` in numerous places (111 occurrences found). Probably better to use `dataTestid` as value throughout to avoid this error-prone mistake